src/kryptos/k4/masking_v2.py

```
@@ -21,13 +21,14 @@
-    "NORTHEAST": 26,
-    "BERLIN": 63,
-    "CLOCK": 69,
-}
+# Original crib positions (0-indexed in the 97-char K4).
+#
+# Previously duplicated K4_CRIBS as a separate, hand-maintained literal here
+# (with EAST/NORTHEAST independently drifted 1 position too high -- the same
+# bug fixed in keystream_validator.K4_CRIBS on 2026-09-02, see that module's
+# note). Derived from the already-imported K4_CRIBS instead, so there is only
+# one source of truth to keep correct.
+ORIGINAL_CRIB_POSITIONS: dict[str, int] = {label: pos for label, (_, pos) in K4_CRIBS.items()}
```

Pylint (4.0.7)

[convention] Line too long (94/88)

(C0301)